### PR TITLE
Add a PR template for literature addition

### PR DESCRIPTION
**Pull request by DunklesArchipel**
A checklist for curators and reviewers of added literature which will be included in new PRs.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+For contributors and reviewers.
+
+- [ ] Name PR `Add Literature-Identifier`
+
+**Filename**
+
+- [ ] Is the filename of type `NAME_YEAR_FIRST-WORD_VOLUME_fxy_IDENTIFIER` (where x is the figure number and y the subfigure label, i.e., 2b).
+- [ ] Does the figure number match that of the figure in the article.
+
```
Comment (TheOpenBIts) [inline .github/pull_request_template.md:12]: - [ ] Everything is written in lower case.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Remove typos (or unwanted whitespace), specifically in the title such as `Pt (111)` to `Pt(111)`.
+
+**SVG**
+
+- [ ] Is the curve label short and unambiguous?
+- [ ] Figure label should be of type `Figure: xy` (not `Figure: fxy`).
+- [ ] Is the figure label identical to that in the article.
+- [ ] Comments should be complete sentences ending with a period.
+- [ ] A `tags` text label should be included in the SVG.
+
```
Comment (TheOpenBIts) [inline .github/pull_request_template.md:25]: - [ ] Do the axis labels have the right units?

**Review by TheOpenBIts — COMMENTED** [review]
Added some additional points to take into account for literature addition and reviewing.

**Comment by TheOpenBIts** [comment]
@DunklesArchipel : Looks good so far.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+**YAML**
+
+- [ ] Is the identifier correct.
+- [ ] Comments should be complete sentences ending with a period.
```
[the remark this answers is not in the document]
Comment (DunklesArchipel) [reply .github/pull_request_template.md:29]: This is later on verified against a schema (see #https://github.com/echemdb/metadata-schema/pull/35)

**Comment by DunklesArchipel** [comment]
> @DunklesArchipel : Looks good so far.

Could you also approve the PR? :)
You need to start a review and approve.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+
+# Adding Literature Checklist (delete if not applicable)
+
+For contributors and reviewers alike.
+
+- [ ] Name PR `Add Literature-Identifier`
```
Comment (DunklesArchipel) [inline .github/pull_request_template.md:6]:
```suggestion
- [ ] Name PR `Add literature_identifier`
```

**Pull request merged**